sklearn/cluster/_k_means_elkan.pyx

```diff
@@ -18,7 +18,7 @@ from ..metrics import euclidean_distances
 from ._k_means import _centers_dense
 
 
-cdef floating euclidian_dist(floating* a, floating* b, int n_features) nogil:
+cdef floating euclidean_dist(floating* a, floating* b, int n_features) nogil:
```

Comment:
for cdef functions, I believe if there is no `pxd` it's basically private, because it cannot be imported from third party projects. @ogrisel / @amueller / @GaelVaroquaux am I right?

Maybe we should make it explicit by adding an underscore too.